src/uptane/tufrepository.cc

```diff
@@ -183,15 +183,6 @@ void TufRepository::initRoot(const Json::Value& new_content) {
                                                                << requiredThreshold);
       throw IllegalThreshold(name_, "root.json contains a role that requires too many signatures");
     }
-    for (Json::ValueIterator it_keyid = (*it)["keyids"].begin(); it_keyid != (*it)["keyids"].end(); ++it_keyid) {
```

Comment:
The TUF/ Uptane guys are trying to merge the two specs to be closer to each other, and this is one place where they diverge slightly. Following TUF definitely means we're following Uptane (in this case).